frontend/WikiContrib-Frontend/src/result.js

```diff
@@ -75,8 +75,8 @@ const chartOptions = {
 class DisplayUser extends React.Component {
   render = () => {
     let { start_time: st, end_time: et } = this.props.filters;
-    st = new Date(st);
-    et = new Date(et);
+    st = new Date(st); let st_m = st.getUTCMonth();
+    et = new Date(et);	    et = new Date(et); let et_m = et.getUTCMonth();
```

Comment:
Twice we have `et = new Date(et);` here. It would be nice to have these statements one in each line.

Reply:
Another way to re-write them would be:
```
let st = new Date(st),
    st_m = st.getUTCMonth(),
    et = new Date(et),
    et_m = et.getUTCMonth();
````

Reply:
@srish  Thank you ma'am. Made the suggested changes. Please review.